### PR TITLE
13.1.0

### PR DESCRIPTION
**Pull request by lisicky**
Changes:

- Fixed json schema generator
- Added `.as_address()` for `PlutusData` type 
- `PlutusData.from_address()` return error for a reward address, because otherwise it might lead misinterpretation of an address.

### CODE REVIEW

**Review by lisicky — COMMENTED** [review]
/check

**Review by lisicky — COMMENTED** [review]
/check

File: rust/json-gen/src/main.rs [hunk 1]
```diff
@@ -7,6 +7,7 @@ use cardano_serialization_lib::*;
 macro_rules! gen_json_schema {
     ($name:ident) => {
         //let out_dir = std::env::var_os("OUT_DIR").expect("no env");
+        println!("Generating schema for {}", stringify!($name));
```
Comment (vsubhuman) [inline rust/json-gen/src/main.rs:10]: Is this intentional?

Comment (lisicky) [reply rust/json-gen/src/main.rs:10]: Yep

**Review by lisicky — COMMENTED** [review]
/check

**Pull request merged**